Xamarin.Forms.Build.Tasks/SetPropertiesVisitor.cs

```diff
@@ -312,6 +318,359 @@ internal static string GetContentProperty(TypeReference typeRef)
 			}
 		}
 
+		//Once we get compiled IValueProvider, this will move to the BindingExpression
```

Comment:
the 340 lines below do not need any comment at all, you should be able to figure what it does, and that what it does is correct, without any effort at all